src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -441,24 +299,42 @@ const WalletTable = ({
                 }}
               >
                 <Td lineHeight="revert">
-                  <Flex justifyContent="space-between" alignItems="center">
-                    <FlexInfo w={{ base: "100%", md: "auto" }}>
+                  <Flex
+                    justifyContent="space-between"
+                    alignItems="center"
+                    w="100%"
+                  >
+                    <FlexInfo
+                      w={{ base: "100%", md: "auto" }}
+                      ps={{ base: 0, md: 1.5 }}
+                    >
                       {/* Wallet image */}
-                      <Box w="56px">
+                      <Box
+                        w={{ base: "24px", md: "56px" }}
+                        mb={deviceLabels.length > 0 ? 0 : 36}
+                      >
                         <Image
                           src={wallet.image}
                           alt=""
                           objectFit="contain"
-                          boxSize="56px"
+                          boxSize="auto"
                         />
                       </Box>
 
                       <Box w={{ base: "100%", md: "auto" }}>
-                        <Stack gap={5}>
-                          <Text>{wallet.name}</Text>
+                        <Stack>
+                          <Box mb={3}>
+                            <Text lineHeight={1.2} fontSize="xl !important">
+                              {wallet.name}
+                            </Text>
+                          </Box>
```

Comment:
> > > ![Screenshot 2024-02-29 at 17 12 31](https://private-user-images.githubusercontent.com/100724231/308917039-1dfb408d-60a6-49c0-8eb6-f0e9f94c5f83.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3MjkyNjYsIm5iZiI6MTcwOTcyODk2NiwicGF0aCI6Ii8xMDA3MjQyMzEvMzA4OTE3MDM5LTFkZmI0MDhkLTYwYTYtNDljMC04ZWI2LWYwZTlmOTRjNWY4My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwNlQxMjQyNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNzM3MjhjZDE3ODEzYmFiMjNhM2I2NzFjYmVkOTEyNTViOGZiMjQ3MTUwMmE5ZTIzOGMzZTkyMjdmMzRhOWNlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.XFXARWM7gG1k0hxwaVVXWkDD15Lnh8ATZO8NJAFTN6A)
> > > there is a too much empty space between tags and wallet name
> > 
> > 
> > unsure if it has been tackled yet, but still an issue
> 
> I'm using the same padding as the 'Visit website' button there

Looks like there is a `gap` being used in this flex box, but then we also have a margin-bottom of 12px (chakra `3`)... perhaps we can just remove this margin?

<img width="749" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/abc9c599-f3c9-4cbd-87b8-0481835e7a21">

```suggestion
                          <Text lineHeight={1.2} fontSize="xl !important">
                            {wallet.name}
                          </Text>
```

Removing the container Box/margin displays as:

<img width="528" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/42d1462f-4a55-4980-afaf-2c6799a4734a">

cc: @konopkja

Reply:
lgtm